Source/Scene/Scene.js

```diff
@@ -1124,9 +1124,20 @@ define([
         if (command._dirty) {
             command._dirty = false;
             
+            var context = scene._context;
+            var derivedCommands = command.derivedCommands;
+
+            if (defined(scene.shadowMap)) {
```

Comment:
Here as well check for `scene.shadowMap.enabled`. Some of this work can wait for multiple shadow maps support.